routes/global_challenge.py

```diff
@@ -24,18 +26,19 @@ def get_individual_saving(meter_id):
     """
 
     # pylint: disable=line-too-long
```

Comment:
Please remove this pragma - it refers to the line below. If the next line is not in conflict with the linter restriction, it is not needed any more.